src/App.vue

```diff
@@ -4,17 +4,30 @@ import BottomNavigation from './components/BottomNavigation.vue'
 import SideBar from './components/SideBar.vue'
 import Header from './components/TheHeader.vue'
 import { isMobile } from './utilities/helper'
+import { useRoute } from 'vue-router'
+import { computed } from 'vue'
+
+const route = useRoute()
+const isHomePage = computed(() => route.path === '/home')
```

Comment:
We need to display side bar and bottom navigation on all the pages except for login. 
Not just on home page.